examples/cr-events.rs

```diff
@@ -0,0 +1,128 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::time::Instant;
+
+use clap::{App, Arg, ArgMatches};
+use colored::*;
+use env_logger;
+
+use microvmi::api::{CrType, EventReplyType, EventType, InterceptType, Introspectable};
+
+fn parse_args() -> ArgMatches<'static> {
```

Comment:
Why the static life time here?

Reply:
`ArgMatches` demands a lifetime:
https://docs.rs/clap/2.33.0/clap/struct.ArgMatches.html

specifying a static one is an accepted solution:
https://users.rust-lang.org/t/cli-how-to-return-clap-type-properly/23906